pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -64,6 +64,28 @@ servingInfo:
   keyFile: /var/serving-cert/tls.key
 providers:
   statuspageID: id-1234
+`
+	finalCMWithCustomValues = `kind: ConsoleConfig
+apiVersion: console.openshift.io/v1
+auth:
+  clientID: console
+  clientSecretFile: /var/oauth-config/clientSecret
+  logoutRedirect: ""
+  oauthEndpointCAFile: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+clusterInfo:
+  consoleBaseAddress: https://` + host + `
+  consoleBasePath: ""
+  masterPublicURL: ` + mockAPIServer + `
+customization:
+  branding: ` + DEFAULT_BRAND + `
+  customLogoFile: custom-logo-file
```

Comment:
I'd use a more realistic value like

```suggestion
  customLogoFile: /var/logo/logo.svg
```